docs/source/cn/array_ops.py

```diff
@@ -732,3 +732,197 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.load,
+    r"""加载一个被 oneflow.save() 保存的对象。
+
+    参数：
+        - **path** (str): 对象所在的路径
+        - **consistent_src_rank** (int, optional): 加载相容 (consistent) 张量的需要的秩 (rank)。被指定时，只有秩与 consistent_src_rank相等的进程才会真正读取 `path` 中的文件，并且被加载对象中的张量会与  placement = `flow.placement('cuda', [consistent_src_rank])` 一致。
+
+    返回类型：
+        加载好的对象
+    """
+)
+
+reset_docstr(
+    oneflow.numel,
+    r"""
+    numel(input) -> int
+
+    返回 :attr:`input` 张量中的元素总数量。
+
+    参数：
+        - **input** (oneflow.Tensor): 输入张量
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+
+        >>> a = flow.randn(1, 2, 3, 4, 5)
+        >>> flow.numel(a)
+        120
+        >>> a = flow.zeros(4,4)
+        >>> flow.numel(a)
+        16
+    """
+)
+
+reset_docstr(
+    oneflow.gather_nd,
+    r"""
+    oneflow.gather_nd(input, index) -> Tensor
+    
+    该算子为 `gather` 的高维扩展， `index` 是一个k维张量，以作为输入张量 `input` 的索引。
+    每个元素将定义一个 `input` 的切片：
+
+    .. math::
+
+        output[i_{0},i_{1},...,i_{K-2}] = input[index(i_{0},i_{1},...,i_{K-2})]
+
+
+    参数：
+        - **input**: 输入张量
+        - **index**: 切片索引
+
+    示例：
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+        >>> input = flow.tensor([[1, 2,3], [4, 5,6],[7,8,9]], dtype=flow.float)
+        >>> index_1 = flow.tensor([[0], [2]], dtype=flow.int)
+        >>> out_1 = flow.gather_nd(input,index_1)
+        >>> print(out_1.shape)
+        oneflow.Size([2, 3])
+        >>> out_1
+        tensor([[1., 2., 3.],
+                [7., 8., 9.]], dtype=oneflow.float32)
+        >>> index_2 = flow.tensor([[0,2], [2,1]], dtype=flow.int)
+        >>> out_2 = flow.gather_nd(input,index_2)
+        >>> out_2
+        tensor([3., 8.], dtype=oneflow.float32)
+
+    """,
+)
+
+reset_docstr(
+    oneflow.grad_enable,
+    r"""
+    启用梯度计算的上下文管理。
+
+    启用梯度计算，如果其被 no_grad 禁用。
```

Comment:
```suggestion
    启用梯度计算，如果其被 no_grad 禁用。
```

这一句的翻译腔太浓厚了，是不是想表达的是 ”如果因为调用了 no_grad 而禁用了梯度计算，那么使用 grad_enable 可以重新启用梯度计算“

为了避免翻译腔，我们可以不句句直译。

Reply:
已修改